packages/blockchain-extension/extension/cloud-account-api.d.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+import { URL } from "url";
+
+/**
+ * The external interface exposed by the IBM Cloud Account extension
+ * for use by other modules to interact with IBM Cloud APIs.
+ */
+export interface CloudAccountApi {
```

Comment:
Might be worth us creating an `interfaces` directory and putting this file in there - just in case we add more in the future.

Reply:
Will do.